test/integration/test-amp-analytics.js

```diff
@@ -74,6 +79,9 @@ describe('amp-analytics', function() {
           // cookie set via http response header when requesting
           // localhost:9876/amp4test/compose-doc
           expect(q['cookie']).to.equal('test');
+          expect(q['fcp']).to.not.be.null;
```

Comment:
Does the test work in Firefox and Safari? I'm quite surprised that the integration test passed 🤔

Reply:
Cleaned up integration tests and tested for all browsers (except Opera, where I only did manual tests).